src/components/sbb-checkbox-group/sbb-checkbox-group.e2e.ts

```diff
@@ -1,73 +1,100 @@
-import { E2EElement, E2EPage, newE2EPage } from '@stencil/core/testing';
+import { assert, expect, fixture } from '@open-wc/testing';
+import { html } from 'lit/static-html.js';
+import { SbbCheckboxGroup } from './sbb-checkbox-group';
+import { SbbCheckbox } from '../sbb-checkbox';
+import { waitForLitRender } from '../../global/testing';
 
 describe('sbb-checkbox-group', () => {
-  let element: E2EElement, page: E2EPage;
+  let element: SbbCheckboxGroup;
 
   beforeEach(async () => {
-    page = await newE2EPage();
-    await page.setContent(`
+    element = await fixture(html`
       <sbb-checkbox-group>
         <sbb-checkbox id="checkbox-1" value="checkbox-1">Label 1</sbb-checkbox>
         <sbb-checkbox id="checkbox-2" disabled value="checkbox-2">Label 2</sbb-checkbox>
         <sbb-checkbox id="checkbox-3" value="checkbox-3">Label 3</sbb-checkbox>
       </sbb-checkbox-group>
     `);
-    element = await page.find('sbb-checkbox-group');
   });
 
   it('renders', async () => {
-    expect(element).toHaveClass('hydrated');
+    assert.instanceOf(element, SbbCheckboxGroup);
+    assert.instanceOf(
+      document.querySelector('sbb-checkbox-group > sbb-checkbox#checkbox-1'),
```

Comment:
Since you are querying for the three `sbb-checkbox` in each test, assign a variable inside the beforeEach for each of them.
Nitpick, I would also shorten the query to just `element.querySelector('#checkbox-1'),`